Kinvey/Kinvey/Client.swift

```diff
@@ -256,6 +279,51 @@ open class Client: Credential {
         }
     }
     
+    /**
+     Initialize a `Client` instance.
+     - parameters:
+       - appKey: `App Key` value from Kinvey Console
+       - appSecret: `App Secret` value from Kinvey Console
+       - instanceId: Prefix value of `Host URL` from Kinvey Console. eg: my-instance if the Host URL looks like my-instance.kinvey.com
```

Comment:
the Host URL looks like `my-instance-baas.kinvey.com`, I think.

Reply:
then the code itself is wrong, let me double check with our infra guys